### ISSUE TITLE
Accuracy calculation in test has redundant instances

### ISSUE THREAD

**Issue by theartpiece** [MCG-NJU/VideoMAE#46]
Under this section [Finetune]( https://github.com/MCG-NJU/VideoMAE/blob/main/FINETUNE.md#original-implementation), it's written that during test, you consider multiple segments and multiple crops
```
    --test_num_segment 2 \
    --test_num_crop 3 \

```
But while calculating accuracy, we don't aggregate accuracy scores over all these segments/crops--
https://github.com/MCG-NJU/VideoMAE/blob/main/engine_for_finetuning.py#L180
https://github.com/rwightman/pytorch-image-models/blob/master/timm/utils/metrics.py#L25

`        acc1, acc5 = accuracy(output, target, topk=(1, 5))
`
Instead while calculating accuracy we should have done something like this--
```
scores=pd.dataFrame({ "id" : ids, "outputs" : outputs, "labels" : labels })
scores=scores.groupby([ "ids" , "labels" ]).aggregate({"outputs": lambda x : max(x)})

```

**Comment by yztongzhan** [comment]
Hi @theartpiece, Thanks for your comments. We use multi-clip testing and the following function is used to merge the scores:
https://github.com/MCG-NJU/VideoMAE/blob/418d3d695365fcb658d003998b6a32a51a1f0d86/engine_for_finetuning.py#L233
